src/data_source/storage/sql/db.rs

```diff
@@ -37,4 +37,8 @@
 /// {}
```

Comment:
I think this big long comment is a bit out of date now that we found a better way of doing it